edx_proctoring/views.py

```diff
@@ -611,18 +611,16 @@ class StudentProctoredExamAttemptsByCourse(AuthenticatedAPIView):
     def get(self, request, course_id, search_by=None):  # pylint: disable=unused-argument
         """
         HTTP GET Handler. Returns the status of the exam attempt.
+        Course and Global staff can view both timed and proctored attempts.
```

Comment:
`Course staff and Global staff can view both timed and proctored exam attempts.`